pkgs/development/python-modules/outdated/default.nix

```
@@ -0,0 +1,37 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  littleutils,
+  requests,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "outdated";
+  version = "0.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "alexmojaki";
+    repo = "outdated";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-5VpPmgIcVtY97F0Hb0m9MuSW0zjaUJ18ATA4GBRw+jc=";
+  };
+
+  build-system = [ setuptools-scm ];
+
+  dependencies = [
+    littleutils
+    requests
+  ];
+
```

`outdated` also have tests but with `unittest` library (https://github.com/alexmojaki/outdated/blob/master/tests.py)

```suggestion
  nativeCheckInputs = [
    unittestCheckHook
  ];
```

---

actually, all the tests rely on internet connection

---

Ok, maybe then use `doCheck = false;`